x/ibc/02-client/client/cli/query.go

```diff
@@ -88,85 +88,11 @@ $ %s query ibc client state [client-id]
 // the chain as defined in https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#query
 func GetCmdQueryConsensusState(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "consensus-state [client-id]",
-		Short:   "Query the latest consensus state of the client",
-		Long:    "Query the consensus state for a particular light client",
-		Example: fmt.Sprintf("%s query ibc client consensus-state [client-id]", version.ClientName),
-		Args:    cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			cliCtx := context.NewCLIContext().WithCodec(cdc)
-			clientID := args[0]
-			if strings.TrimSpace(clientID) == "" {
-				return errors.New("client ID can't be blank")
-			}
-
-			prove := viper.GetBool(flags.FlagProve)
-
-			csRes, err := utils.QueryConsensusState(cliCtx, clientID, prove)
-			if err != nil {
-				return err
-			}
-
-			return cliCtx.PrintOutput(csRes)
-		},
-	}
-	cmd.Flags().Bool(flags.FlagProve, true, "show proofs for the query results")
-	return cmd
-}
-
-// GetCmdQueryRoot defines the command to query a verified commitment root
-func GetCmdQueryRoot(queryRoute string, cdc *codec.Codec) *cobra.Command {
```

Comment:
This isn't needed?

Reply:
it was moved as a field for the consensus state